### DISCUSSION TITLE
I got locked out after installing Spatie Permissions

### DISCUSSION THREAD

**Discussion by wordxpression** [spatie/laravel-permission#2611] [Q&A]
I installed Spatie permissions, Generated the required permissions for the superadmin role... but got locked out and can't login anymore. 

For sure it's something small, but I can't find it.

**Comment by drbyte** [comment]
Without seeing your code it's really hard to give you a specific answer.
The documentation explains how to install, and how to set up permissions, and how to adapt your application code to check for permissions in order to authorize access to whatever parts of your application you choose to add protection for.

Also: This package has NOTHING to do with login. That's "authentication". This package only deals with "authorization" after login.
